assets/wizards/advertising/views/placements/index.js

```diff
@@ -64,90 +64,124 @@ class Placements extends Component {
 
 		return (
 			<Fragment>
-				<h2>{ __( 'Pre-defined ad placements' ) }</h2>
-				<p>
-					{ __(
-						'Define global advertising placements to serve ad units on your site. Enable the individual pre-defined ad placements to select which ads to serve.'
+				<SectionHeader
+					title={ __( 'Pre-defined ad placements', 'newspack' ) }
+					description={ () => (
+						<>
+							{ __(
+								'Define global advertising placements to serve ad units on your site',
+								'newspack'
+							) }
+							<br />
+							{ __(
+								'Enable the individual pre-defined ad placements to select which ads to serve',
+								'newspack'
+							) }
+						</>
 					) }
-				</p>
-				<ToggleGroup
+				/>
+				<ActionCard
+					isMedium
 					title={ __( 'Global: Above Header' ) }
 					description={ __( 'Choose an ad unit to display above the header' ) }
-					checked={ global_above_header && global_above_header.enabled }
-					onChange={ value => togglePlacement( 'global_above_header', value ) }
+					toggleChecked={ global_above_header && global_above_header.enabled }
+					hasGreyHeader={ global_above_header && global_above_header.enabled }
+					toggleOnChange={ value => togglePlacement( 'global_above_header', value ) }
 				>
-					<AdPicker
-						adUnits={ adUnits }
-						services={ services }
-						value={ global_above_header }
-						onChange={ value => onChange( 'global_above_header', value ) }
-					/>
-				</ToggleGroup>
-				<ToggleGroup
+					{ global_above_header && global_above_header.enabled ? (
+						<AdPicker
+							adUnits={ adUnits }
+							services={ services }
+							value={ global_above_header }
+							onChange={ value => onChange( 'global_above_header', value ) }
+						/>
+					) : null }
+				</ActionCard>
+				<ActionCard
+					isMedium
 					title={ __( 'Global: Below Header' ) }
-					description={ __( 'Choose an ad unit to display above the header' ) }
-					checked={ global_below_header && global_below_header.enabled }
-					onChange={ value => togglePlacement( 'global_below_header', value ) }
+					description={ __( 'Choose an ad unit to display below the header' ) }
```

Comment:
Missing i18n domain - here and in few other spots in this file:

```suggestion
					description={ __( 'Choose an ad unit to display below the header', 'newspack' ) }
```